src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/SaveTempDataAttribute.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.AspNetCore.Mvc.ViewFeatures
     public class SaveTempDataAttribute : Attribute, IFilterFactory, IOrderedFilter
     {
         /// <inheritdoc />
-        public int Order { get; set; }
+        public int Order { get; set; } = int.MinValue + 100;
```

Comment:
Document why it needs to be this way - what are you trying to avoid by setting an explicit order?